### PR TITLE
Cornwell's 30-39.

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
This is @rcornwell's second transcription of pages 30-39.

### CODE REVIEW

File: transcribed/page0300.txt
Comment (aap) [inline transcribed/page0300.txt:1]: changes are mistakes

File: transcribed/page0310.txt [hunk 1]
```diff
@@ -9,46 +9,46 @@
...
+036    
 037     118 036 AGETMSK:        MOVE A,MSKST(U) ;OPER 5
 038     001 006         MOVEM A,(J)
 039     001 016         POPJ P,
-040
-041     001 006 ASETMSK:        MOVE A,(J)      ;OPER 4
-042     001 025         TRZ A,BPIPT+VALRTF+BRKF+BCNTRZI ;MAKE SURE THESE GO TO SUP. PRO.
+040    
+041     001 006 ASETMSK:        MOVE A,(J)     ;OPER 4
+042     001 025         TRZ A,BPIPI+VALRTF+BRKF+BCNTRZI ;MAKE SURE THESE GO TO SUP. PRO.
```
Comment (aap) [inline transcribed/page0310.txt:42]: this is correct. the other diffs on this page are mistakes

File: transcribed/page0320.txt [hunk 2]
```diff
@@ -37,19 +37,19 @@
...
-040
+040    
 041             ATRNDL:         ;CALL 7,
 042     117 028         MOVE A,SIOMTU
 043     001 007         MOVE B,(C)
 044     118 044         CAME B,UNAME(U)
 045     001 016         POPJ P,
 046             TRND:   IRPC N,,12345678
-047     001 007         MOVE B,TRAN1N(A)
+047     001 007         MOVE B,TRANIN(A)
```
Comment (aap) [inline transcribed/page0320.txt:47]: i belive this is TRAN!N, also in line 35

File: transcribed/page0320.txt [hunk 3]
```diff
@@ -10,10 +10,10 @@
 010     001 016         POP P,1(C)
 011     001 016         POP P,(C)
 012     100 023         JRST POPJ1
-012
+013
```
Comment (aap) [inline transcribed/page0320.txt:13]: correct, the other diffs on this page are mistakes

File: transcribed/page0330.txt [hunk 4]
```diff
@@ -10,40 +10,40 @@
...
 026     118 046         HRRZ A,TTYTBL(A)
 027     033 014         JRST AATT4
-028
+028    
 029     118 046 AATT3:  HLL A,TTYTBL(A)
 030     001 006         TLZ A,700000
 031     070 028         MOVEM A,TTYSTS(B)
-032     118 046         HRRRZM B,TTYTBL(A)
-033     001 048 CLK0J1: CONO PI,CLKON
+032     118 046         HRRZM B,TTYTBL(A)
```
Comment (aap) [inline transcribed/page0330.txt:32]: these two are correct, the others on this page are mistakes

File: transcribed/page0331.txt [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
-072
+072    
 073     118 008 AIPUSH: MOVE T,SIOCP(U) ;OPER 13
 074     001 017         HLRZ A,T
 075     001 006         CAIL A,20
 076     021 049         JRST IOCERR
 077     001 017         PUSH T,(R)
 078     118 008         MOVEM T,SIOCP(U)
-079
+079                     CLEARM (R)
```
Comment (aap) [inline transcribed/page0331.txt:19]: correct, others mistakes

File: transcribed/page0340.txt [hunk 6]
```diff
@@ -1,8 +1,8 @@
-001     019 015 SUBTTL CORE ALLOCATOR                                           PAGE 34
-002
+001     019 015 SUBTTL CORE ALLOCATOR                                       PAGE 34
+002    
 003             UACORE:
-004     117 035 ACORE:  CAILE B,MXC     ;U HAS USER THAT WANTS CORE
-005     001 016         POPJ P,         ;IF U=USER,THIS IS IN CORE
+004     117 035 ACORE:  CAILE B,MXC     ;U HAS USER THAT WANTS CODE
```
Comment (aap) [inline transcribed/page0340.txt:4]: mistake

File: transcribed/page0340.txt [hunk 7]
```diff
@@ -1,8 +1,8 @@
...
+005     001 016         POPJ P,         ;IF U=USER,THIS GUY IS IN CORE
```
Comment (aap) [inline transcribed/page0340.txt:5]: correct

File: transcribed/page0340.txt [hunk 8]
```diff
@@ -15,17 +15,17 @@
 015     118 029         MOVSI A,BUCSTP
 016     118 027         IORM A,USTP(U)
 017     001 048         CONO PI,CLKON
-018     118 058 ACR1A:  MOVEM B,EPOL(U)
+018     118 058 ACR1A:  MOVEM B,EPDL(U)
```
Comment (aap) [inline transcribed/page0340.txt:18]: correct

File: transcribed/page0340.txt [hunk 9]
```diff
@@ -15,17 +15,17 @@
...
 019     018 020 ACORE1: PUSHJ P,SWAIT
 020     117 027         CRACF
 021     117 013         CAMN U,DISUSR
 022     036 002         PUSHJ P,DCRSTP
-023     117 012         MOVEM U,CUSER
+023     117 012         MOVEM U,DUSER
```
Comment (aap) [inline transcribed/page0340.txt:23]: mistake

File: transcribed/page0340.txt [hunk 10]
```diff
@@ -15,17 +15,17 @@
...
 024     051 054         SETOM MEMFRZ
 025     034 028         JUMPN B,ACR1B
 026     118 041         SKIPN MEMTOP(U)
 027     038 014         JRST ACORE9     ;HAS NO MEMORY WANTS NO MEMORY
-028     118 041 ACRIB:  SKIPN A,MEMTOP(U)
+028     118 041 ACR1B:  SKIPN A,MEMTOP(U)
```
Comment (aap) [inline transcribed/page0340.txt:28]: correct

File: transcribed/page0350.txt [hunk 11]
```diff
@@ -1,10 +1,10 @@
-001                                                                             PAGE 35
-002     001 006 ACORE2: MOVEI C,1(A)
+001                                                                   PAGE 35
+002     001 008 ACORE2: MOVEI C,1(A)
```
Comment (aap) [inline transcribed/page0350.txt:2]: correct

File: transcribed/page0350.txt [hunk 12]
```diff
@@ -13,16 +13,16 @@
...
+018     002 002         MOVSI D,-MX1KN  ;INITITIATE MOBY SHUFFLE
 019     001 012         MOVEI I,0
 020     037 019         JRST ACMS2
-021
+021    
 022     016 002 ACRF:   PUSHJ P,IODCL
 023     001 007         MOVEI B,0
 024     034 008         PUSHJ P,ACRF1
-025                     JCFL
+025                     JFCL
```
Comment (aap) [inline transcribed/page0350.txt:25]: correct, rest mistakes

File: transcribed/page0360.txt [hunk 13]
```diff
@@ -13,12 +13,12 @@
...
 015     001 016         POPJ P,
-016
-017     038 013 ACORE4: JRST ACR9A      ;TEMP
+016    
+017     038 013 ACORE4: JSRT ACR9A      ;TEMP
 018     002 022         CONO PI,UTCOFF
 019     083 051         MOVE D,UTTBF
 020     001 009         CAIGE D,8
-021     038 012         JRST ACR9B      ;CANNOT GET ANOTHER BLOCK BY RECLAIMING FREE IO BUFFER
+021     038 012         JRST ACR9B      ;CANNOT GET ANOTHER BLOCK BY RECAIMING FREE IO BUFFER
```
Comment (aap) [inline transcribed/page0360.txt:21]: correct, in a way :)

File: transcribed/page0310.txt (hunk 1 above)
Comment (larsbrinkhoff) [reply transcribed/page0310.txt:42]: Yes, BPIPI is a good catch.

File: transcribed/page0370.txt [hunk 14]
```diff
@@ -1,23 +1,23 @@
-001                                                                             PAGE 37
+001                                                    PAGE 37
 002     001 009 ACORE6: MOVE A,D
 003     045 026         PUSHJ P,CCFRE
 004     051 039         DPB A,[MLO,,MEMBLT-1(A)
 005     001 008         MOVEI C,377
 006     051 039         DPB C,[MLO,,MEMBLT(A)
-007     001 009         SOS A
+007     001 006         SOS A
```
Comment (aap) [inline transcribed/page0370.txt:7]: correct

File: transcribed/page0370.txt [hunk 15]
```diff
@@ -1,23 +1,23 @@
...
 013     118 024 ACORTN: HLRZ C,UPR(U)
 014     118 041         SKIPN MEMTOP(U)
 015     001 008         MOVNI C,2000
 016     001 008         ADD A,C
 017     044 055         JRST ACORT1
-018
+018    
 019     001 009 ACMS2:  MOVE J,D        ;D HAS FIRST FREE BLOCK BELOW
-020     051 039 ACMS7:  LDB E,[MUR,,MEBLT(J)
+020     051 039 ACMS7:  LDB E,[MUR,,MEMBLT(J)
```
Comment (aap) [inline transcribed/page0370.txt:20]: correct

File: transcribed/page0380.txt
Comment (aap) [inline transcribed/page0380.txt:1]: all diffs mistakes

File: transcribed/page0390.txt [hunk 16]
```diff
@@ -1,39 +1,39 @@
...
-004     014 008         PUSHJ P,UADRS2  ;MOVE USER VARIABLES
-005     040 002 ACMS4G: JSP Q,ACMS4B    ;WRITING POINTER
+001                                                                   PAGE 39
+002     039 005 ACMS4D: JUMPN I,ACMS4G   ;JUMP ON MOVEING DOWN
+003     001 009         HRRZ A,D         ;MOVE USER
+004     014 008         PUSHJ P,UADRS2   ;MOVE USER VARIABLES
+005     040 002 ACMS4G: JSP Q,ACMS4B     ;WRITING POINTER
 006     039 013 ACMS5:  ADD D,ACMDT2(I)
-007     037 020 ACMS8:  XCT ACMDT3(I)
+007     039 015 ACMS8:  XCT ACMDT3(I)
```
Comment (aap) [inline transcribed/page0390.txt:7]: correct, rest mistakes

File: transcribed/page0320.txt (hunk 2 above)
Comment (larsbrinkhoff) [reply transcribed/page0320.txt:47]: Ah, tricky one!  Yes, those two are ! which is valid inside an IRP (or macro).

File: transcribed/page0360.txt (hunk 13 above)
Comment (larsbrinkhoff) [reply transcribed/page0360.txt:21]: I like the original typos!

File: transcribed/page0350.txt (hunk 12 above)
Comment (larsbrinkhoff) [reply transcribed/page0350.txt:25]: EXECPT and MOVSI D, look correct.

**Pull request merged**